packages/@uppy/core/src/index.js

```diff
@@ -979,6 +979,13 @@ class Uppy {
       }
     })
 
+    // If file is a google doc uploaded by companion
+    // file.size is null
+    if (data.bytesTotal === data.bytesUploaded && !file.size) {
```

Comment:
some nitpicks 😇 
you've actually addressed a broader issue here, as photos from instagram and facebook also don't have a size. so we can be a bit more accurate in the comment by mentioning remote providers in general:

```suggestion
    // Remote providers sometimes don't tell us the file size,
    // but we can know how many bytes we uploaded once the upload is complete.
    if (data.bytesTotal === data.bytesUploaded && file.size == null) {
```

i'd also change the `!file.size` to `file.size == null`: we care about unknown file sizes here, not 0-sized files, best to be explicit about it.